src/api/code-challenge/content-types/code-challenge/test-evaluator.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  evaluateWithContext,
+  getCode,
+  getEvaluationContext,
+  overrideConsoleLog,
+  restoreConsoleLog,
+} from "../code-challenge/utils";
+
+type runTestEvaluatorProps = {
+  metaCaseCode: string;
+  internalTestCode: string;
+  metaLabel: string;
+  challengeLabel: string;
+  metaTestId: number;
+  evalResultShouldBe: boolean;
+  internalTestId: number;
+  removeComments?: boolean;
+};
+
+export const runTestEvaluator = async ({
+  metaCaseCode,
+  internalTestCode,
+  metaTestId,
+  evalResultShouldBe,
+  internalTestId,
+  removeComments,
+}: runTestEvaluatorProps) => {
+  const getShortError = () => {
+    return `${evalError.stack
+      ?.toString()
+      .substring(0, 250)} --- (End of Abridged Error)`;
+  };
+
+  const getNoError = () => {
+    return "no eval() error detected";
+  };
```

Comment:
This doesn't need to be a function it can just be a string

Reply:
Thanks again.